CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# suitcase-csv changes
+
+v0.2.1 (2019-08-16)
```

Comment:
Normally these would titles (with a `##` on the left of the line) with the changes listed underneath. It's OK to leave these sections empty for now---retroactively writing release notes for old releases is probably too low priority to be worth doing at all---but I think that should be formatted as titles anyway, perhaps with "TO DO" underneath, or left blank.